xbmc/video/videosync/VideoSyncCocoa.cpp

```diff
@@ -133,6 +133,7 @@ void CVideoSyncCocoa::UpdateFPS(double fps)
   {
     CLog::Log(LOGDEBUG, "CVideoSyncCocoa: Detected refreshrate: %i hertz", fpsInt);
     m_fps = fpsInt;
+    g_VideoReferenceClock.UpdateRefreshrate();
```

Comment:
I thought probably not, but I didn't know any better and it felt like the best way to get the ball going was to at least show an example patch that fixed the problem :)